src/lesson6_task1/LightTransport.java

```diff
@@ -0,0 +1,56 @@
+package lesson6_task1;
+
+
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+public class LightTransport extends LandTransport {
+    String bodyworkType;
+    int countPassengers;
+    float way;
+
+    //конструктор
+    public LightTransport(int powerHours, int maxSpeed, String brand, float weight, int countOfWheels, float fuelConsumption,
+                          String bodyworkType, int countPassengers) {
+        super(powerHours, maxSpeed, brand, weight, countOfWheels, fuelConsumption);
+        this.bodyworkType = bodyworkType;
+        this.countPassengers = countPassengers;
+        this.way = way;
```

Comment:
@Nadya101 И снова 
```suggestion
        this.way = someFloatValue;  //например 12,3f
```
ты не передала в параметры конструктора значение way, оно вообще не должно было компилироваться....

Reply:
Удалила все **поля** из конструктора, значение которых **не вносим** при создании экземпляров.